src/main/java/com/sollace/coppergolem/entity/CopperGolemEntity.java

```diff
@@ -51,8 +51,8 @@
 import com.sollace.coppergolem.GItems;
 import com.sollace.coppergolem.GSounds;
 import com.sollace.coppergolem.entity.ai.BlockInteraction;
-import com.sollace.coppergolem.entity.ai.PressButtonGoal;
 import com.sollace.coppergolem.entity.ai.ChaseTargetGoal;
+import com.sollace.coppergolem.entity.ai.PressButtonGoal;
```

Comment:
Ditto for the comment about out-of-order imports in `client/CopperGolemEntityRenderer.java`.